content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -325,9 +326,9 @@ Name:      kubernetes.default
 Address 1: 10.0.0.1 kubernetes.default.svc.cluster.local
 ```
 
-If this fails, you might need to go to the kube-proxy section of this document,
-or even go back to the top of this document and start over, but instead of
-debugging your own Service, debug the DNS Service.
+If this fails, please see the [kube-proxy](#Is-the-kube-proxy-working) section
```

Comment:
Hello @thockin .
I read through the page again. LGTM.
One nit: This new link is not working, update to (not the back ticks): ```[kube-proxy](#is-the-kube-proxy-working)```

Reply:
Changed as request - just a capital letter?